setup.py

```diff
@@ -1,62 +0,0 @@
-#!/usr/bin/env python
-
-# Copyright 2016 OpenMarket Ltd
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from setuptools import setup
-from codecs import open
-import os
-
-here = os.path.abspath(os.path.dirname(__file__))
-
-
-def read_file(path_segments):
-    """Read a UTF-8 file from the package. Takes a list of strings to join to
-    make the path"""
-    file_path = os.path.join(here, *path_segments)
-    with open(file_path, encoding="utf-8") as f:
-        return f.read()
-
-
-def exec_file(path_segments, name):
-    """Extract a constant from a python file by looking for a line defining
-    the constant and executing it."""
-    result = {}
-    code = read_file(path_segments)
-    lines = [line for line in code.split('\n') if line.startswith(name)]
-    exec("\n".join(lines), result)
-    return result[name]
-
-
-setup(
-    name="matrix-synapse-ldap3",
-    url="https://github.com/matrix-org/matrix-synapse-ldap3",
-    version=exec_file(("ldap_auth_provider.py",), "__version__"),
-    py_modules=["ldap_auth_provider"],
-    description="An LDAP3 auth provider for Synapse",
-    install_requires=[
-        "Twisted>=15.1.0",
-        "ldap3>=2.8",
-        "service_identity",
-    ],
-    test_require=[
-        "matrix-synapse",
-    ],
```

Comment:
I don't see this having been copied over, but I think that's fine because this is only part of `python setup.py test` from what I can tell?